addons/barcodes/static/src/components/manual_barcode.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="barcodes.BarcodeInput">
+        <div class="input-group">
+            <input id="manual_barcode" t-model="state.barcode" type="text" name="barcode"
+                    class="form-control" t-ref="manualBarcode"
+                    t-att-placeholder="props.placeholder || 'Enter a barcode...'" t-on-keydown="_onKeydown"/>
```

Comment:
Define the defaultProps in JS and use the translation function. Here, we can't translate "Enter a barcode...".